src/styl/responsive/mobile.styl

```diff
@@ -49,6 +49,7 @@ $main-content-offset = $margin-small + $margin-mid
 
 #main-content
   width: 360px
+  transition: opacity $fade-duration
```

Comment:
This doesn't seem to be true. Testing my end removing this line means only the images transition but the text doesnt